src/lib.rs

```diff
@@ -57,6 +57,9 @@ pub mod mem;
 #[cfg(target_arch = "arm")]
 pub mod arm;
 
+#[cfg(all(target_arch = "aarch64", not(feature = "no-asm"),))]
+pub mod aarch64;
```

Comment:
@nilstrieb found in https://github.com/rust-lang-ci/rust/actions/runs/5433440965/jobs/9881115904#step:24:8768 that this is wrong - it also needs to gate on `target_os = "linux"`